linux_os/guide/system/network/network-firewalld/ruleset_modifications/set_firewalld_default_zone/rule.yml

```diff
@@ -24,6 +24,7 @@ identifiers:
     cce@rhel7: CCE-27349-0
     cce@rhel8: CCE-80890-7
     cce@rhel9: CCE-84023-1
+    cce@sle15: CCE-91410-1
```

Comment:
This ID doesn't come from the `shared/references/cce-sle15-avail.txt` pool. Where does it come from?

Reply:
There is internal repo, based on which my teammate @rumch-se created the cce-sle15-avail.txt and cce-sle12-avail.txt. In it there were several cces that were still not utilized but reserved for the current ongoing effort for pcidss profile. I am guessing since those were reserved, that was the motivation not to be added in the available lists. There I think 4 more "reserved" that are not visible in cce-sle15-avail.txt , if you think it is more appropriate I can add them in the context of this PR, or better create a new one?

Reply:
@teacup-on-rockingchair Thanks for your explanation, if the CCE is valid and it's been assigned before the pool has been submitted to the repository then you don't need to add and remove it from the pool in this PR.